hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -257,14 +258,35 @@ public void testUpsertsPrepped() throws Exception {
         HoodieWriteClient::upsertPreppedRecords, true);
   }
 
+  /**
+   * Test UpsertPrepped API for HoodieRecord's new location tagging during merge handle with
+   * {@link com.uber.hoodie.common.util.collection.ExternalSpillableMap}
+   * Fixes Hudi issue #773.
+   */
+  @Test
+  public void testUpsertsPreppedUsingSpillableDiskForMerge() throws Exception {
```

Comment:
can we please unit test this at the MergeHandle level? these entire upsert tests keep adding to test run time and we just did a large exercise to rein the CI runtime back to control.

Reply:
is there any sample test you have in mind? usage of MergeHandle in a good example com.uber.hoodie.func.TestUpdateMapFunction#testSchemaEvolutionOnUpdate?